tools/PublishToBlobFeed.proj

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildThisFileDirectory)/BuildToolsetVersions.props" />
+  <Import Project="$(MSBuildThisFileDirectory)/obj/BuildToolset.proj.nuget.g.props" />
```

Comment:
Do you need exists checks for these or are you guaranteed to only ever load this after restore?

Reply:
If these are not imported - we cannot find the DLL - they must exist or the target cannot fire.